src/server/game/BlackMarket/BlackMarketMgr.h

```diff
@@ -0,0 +1,166 @@
+/*
+* Copyright (C) 2008-2015 TrinityCore <http://www.trinitycore.org/>
```

Comment:
Like Carbenium said: "Missing space at the beginning of this and the following header lines."
(Lines 2 - 16 need 1 space in front of them to follow the source standard.)